starter/src/test/java/com/starter/fullstack/rest/InventoryControllerTest.java

```diff
@@ -0,0 +1,70 @@
+package com.starter.fullstack.rest;
```

Comment:
Delete this file since the controller methods/tests aren't relevant in task one.